### PR TITLE
Stylelint

### LINKED ISSUE #19: Add stylelint
### Description

- Add stylelint
- Have it run for pull_requests
- Have it extend off of [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard)

### PR DESCRIPTION
**Pull request by Rhoulhac**
### Description
- Adding stylelint
  - closes #19

### CODE REVIEW

**Comment by Rhoulhac** [comment]
closes #19

File: .stylelintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "stylelint-config-standard"
+}
```
Comment (kirbyjs) [inline .stylelintrc]: :snake: no newline

**Comment by kirbyjs** [comment]
> closes #19

@Rhoulhac This needs to be in the PR description, I've fallen to that mistake before 🙃 (I updated it)

**Comment by kirbyjs** [comment]
@Rhoulhac we have to be careful about [this](https://github.com/kirbyjs/500/pull/23/commits/e0165c9c84f548d007830b7ae97f31c0ec0f9d3e#diff-32607347f8126e6534ebc7ebaec4853d), it had the deere npm registry in the lock file 😅

**Pull request merged**